CppCoreGuidelines.md

```diff
@@ -10328,7 +10328,7 @@ In a class template, nonvirtual functions are only instantiated if they're used
 
 * Flag a class template that declares new (non-inherited) virtual functions.
 
-## <a name="SS-tpg-concepts"></a> TPG.concepts: Concept rules
+## <a name="SS-concepts"></a> T.concepts: Concept rules
 
```

Comment:
true but that needs some love as well,
some anchors are
SS-t-foobar
SS-temp-foobar
SS-foobar
in the chapter on templates